src/components/Text/Text.test.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { shallow, render } from 'enzyme';
+import Text, { isParagraph, isHeading, getHeadingSize, selectTag } from './Text';
+
+describe('Checking if type is paragraph', () => {
+  it('for paragraph', () => {
```

Comment:
Tests should be also documentation of code.
```suggestion
describe('isParagraph function', () => {
  it('should valid correct paragraphs', () => {
```